src/coreComponents/mesh/CellElementSubRegion.cpp

```diff
@@ -171,6 +173,87 @@ localIndex CellElementSubRegion::unpackUpDownMaps( buffer_unit_type const * & bu
   return unPackedSize;
 }
 
+localIndex CellElementSubRegion::packFracturedElementsSize( arrayView1d< localIndex const > const & packList,
+                                                            arrayView1d< globalIndex const > embeddedSurfacesLocalToGlobal ) const
```

Comment:
why is one of these a `const &` and one a non-const view? I think views should always be const?